docs/PREFERENCES.md

```diff
@@ -11,13 +11,14 @@ Preferences can be controlled and modified in the settings window or via the `pr
 | titleBarStyle          | String  | custom        | The title bar style on Linux and Window: `custom` or `native`                                                                                              |
 | openFilesInNewWindow   | Boolean | false         | true, false                                                                                                                                                |
 | openFolderInNewWindow  | Boolean | false         | true, false                                                                                                                                                |
-| zoom                   | Number  | 1.0           | The zoom level. Between 0.5 and 2.0 inclusive.                                                                                              |
+| zoom                   | Number  | 1.0           | The zoom level. Between 0.5 and 2.0 inclusive.                                                                                                             |
 | hideScrollbar          | Boolean | false         | Whether to hide scrollbars. Optional value: true, false                                                                                                    |
+| jumpToStartOnLoad      | Boolean | false         | Jump to the start of the document on load (rather than end)                                                                                                |
```

Comment:
I would prefer a enum because we could add additional options like keep last position (with session).